packages/hint-performance-budget/src/hint.ts

```diff
@@ -99,13 +100,18 @@ export default class PerformanceBudgetHint implements IHint {
                 response.body.rawContent.byteLength :
                 response.body.content.length;
             let sentSize: number;
+            const contentEncoding = response.headers && response.headers['content-encoding'] || '';
```

Comment:
I think you should still use `getHeaderValueNormalized` in case you have empty (`<space>`)  header value.

Reply:
done